tests/test_hotness_consumer.py

```diff
@@ -410,6 +411,146 @@ def test_call_anitya_update_all_versions(self):
             package, "update.bug.file", exp_opts
         )
 
+    def test_bugzilla_notify_with_retrieved_versions(self):
```

Comment:
I know this is here in other tests, but you shouldn't run more than one test case in one method. Either split this up or use [pytest parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html).